src/pages/ports.astro

```diff
@@ -42,13 +49,7 @@ const getCategories = (fields: Port | Userstyle) => {
           .map(([slug, fields]) => (
             <PortCard
               title={fields.name}
-              link={
-                "url" in fields
-                  ? (fields.url as string)
-                  : "alias" in fields
-                    ? `https://github.com/catppuccin/${fields.alias || slug}`
-                    : `https://github.com/catppuccin/userstyles/blob/main/styles/${slug}`
-              }
```

Comment:
What was wrong about this?

Reply:
it was taking the last branch in all cases, so all ports linked to userstyles/*

Reply:
Well not in all cases, but yeah I'm now realizing only a few ports have an alias lol